sparta/test/TestingMacros.cmake

```diff
@@ -61,3 +60,15 @@ macro (sparta_recursive_copy build_target cp_file)
     add_custom_command (TARGET ${build_target} PRE_BUILD
       COMMAND cp -r ${CMAKE_CURRENT_SOURCE_DIR}/${cp_file} ${CMAKE_CURRENT_BINARY_DIR}/)
 endmacro (sparta_recursive_copy)
+
+#
+# Create a sparta test executable
+#
+macro (sparta_add_executable target)
```

Comment:
Is this just for tests?  If so, can you name it `sparta_add_test_executable`?  Or, I think it would be better to change the comment to say this is a helper for adding an executable that needs to link against `${Sparta_LIBS}`.  Could be anything, doesn't have to be a test.  If you change the comment, this macro should probably be defined wherever `${Sparta_LIBS}` is defined.

Reply:
Done -- renamed the macro.  Will push the changes in a minute.

I thought about the "anything" concept, but I don't want to force people that use Sparta into a corner where the default args for `add_executable` are not sufficient for their use.